ci-operator/config/openshift/release/openshift-release-master__nightly-4.6.yaml

```diff
@@ -74,16 +74,6 @@ tests:
   steps:
     cluster_profile: equinix-ocp-metal
     workflow: baremetalds-e2e
-- as: e2e-metal-assisted
```

Comment:
maybe it's possible removing ``assisted-*`` images from ``base_images`` as well
can you check that?

Reply:
good catch! it seems it's possible since it's the only assisted job in this file